game/models.py

```
@@ -223,3 +223,20 @@ def save(self, *args, **kwargs):
+
+
+class ChessPuzzle(models.Model):
+    title = models.CharField(max_length=200)
+    fen = models.CharField(max_length=255)
+    solution = models.JSONField(help_text="JSON array of moves representing the solution, e.g. ['g2g4']")
+    difficulty = models.CharField(
+        max_length=20,
+        choices=[("easy", "Easy"), ("medium", "Medium"), ("hard", "Hard")],
+        blank=True,
+        null=True
+    )
```

Ruff (0.15.15)

[warning] Avoid using `null=True` on string-based fields such as `CharField`

(DJ001)